software/Makefile.nvidia

```diff
@@ -10,7 +10,7 @@ endif
 
 # Use our own fork until this PR is merged: https://github.com/NVIDIA/open-gpu-kernel-modules/pull/3
 NV_DRIVER_URL := https://github.com/JuliaComputing/open-gpu-kernel-modules.git
-NV_DRIVER_VERSION := sjk/rebar2
+NV_DRIVER_VERSION := sjk/rebar4
```

Comment:
So it looks like this is rebased upon a newer driver version, and also reverts a commit?  Was that commit bad?  Is this change necessary for this PR?

Reply:
It would be good to have, since it is what has been running locally on pathfinder for months now. I only noticed the branch discrepancy last night when rebuilding. The reverted commit was some requests from Nvidia that I didn't quite get right.

Reply:
Alright.  In the future, let's try to keep changes like this in a separate PR, so that it's easier to revert in the future, if we need to.